designs/virtualization/4.x/Templates/Templates.md

```diff
@@ -0,0 +1,47 @@
+# Templates
+
+## Virtual machines and VM templates are located under Virtualization
+
+One of the options for users to create a VM is when they choose a to create it from a template.
+Under the left side navigation tab ‘Virtualization’ we'll have 2 tabs: ‘Virtual Machine Templates’ and ‘Virtual Machines’.
+
+### Virtual Machine Templates Tab
+
+From the Virtualization left nav item the user goes to the Virtual machine templates tab where they can see the full list of templates.
+From this point, we suggest 2 options to create a VM:
+
+Create VM via an action from the kebab menu and also allow an action link.
+
+![TemplatesTabOp1](img/TemplatesTabsOp1.png)
+
+![TemplatesTabOp2](img/TemplatesTabsOp2.png)
+
+Both options will lead the user to the general step of the wizard where they will need to provide a name for the VM they want to create.
+
+![UserNeedsToProvideName](img/provide-name.png)
+
+If all they need is to provide a name for the VM, they can skip to the 'Review and create' step where they can review the settings and create the VM.
+
+![user reviews settings](img/CompletesReqiredInfoTemplate.png)
```

Comment:
I wonder for the review step if we should standardize on this? I believe this is more close to what we'll have
https://codepen.io/matthewcarleton/pen/VwLVZre
One small nit here: Created at maybe shouldn't be included b/c we can't say when it was created before it's created :)